docs/YIP/YIP-0005-session-key-management.md

```diff
@@ -0,0 +1,57 @@
+# 0005 - Session Key Management for ClearSync
+
+## Status
+
+Work in progress
+
+## Context
+
+The adoption of session keys enhances security and operational efficiency.
+By limiting the validity of keys and associating them with specific counterparts, we significantly reduce risks associated with key exposure and unauthorized access.
+
+This proposal introduces a new strategy for key management in the ClearSync project, shifting from signing sub-keys to registering session keys.
+Each Clearport will generate a key pair limited to a default duration of 24 hours, termed a clearing session.
+
+## Decision
+
+### Key Generation and Registration
+
+- **Generation**: Each Clearport generates a key pair valid for a default duration of 24 hours.
+- **Registration**: Participants must register the clearing public key on the adjudicator, specifying the duration, counterpart, and owner (Smart-Account).
+- **Duration**: 24 hours
+- **Counterpart**: Smart-Account
+- **Owner**: Smart-Account
+
+### Key Features
+
+- **Spending Limit**: A spending limit can be added to the clearing key.
+- **Collateral Association**: The collateral is associated with this clearing key.
+- **Revocation**: The Clearing-Key can be revoked at any time.
+- **NeoDAX Access**: The same key can access NeoDAX, with the trading session having the same duration.
```

Comment:
If the key is revoked but only a key with the same duration can reenter, then both parties have to regenerate the session keys to be at the same duration. Do I understand it correctly? Is it intended to be like this?

Reply:
I'm not sure I understand you question to this line